packages/sanity/src/structure/panes/documentList/DocumentListPaneContent.tsx

```diff
@@ -247,10 +247,6 @@ export function DocumentListPaneContent(props: DocumentListPaneContentProps) {
         </CommandListBox>
       </RootBox>
     )
-    // Explicitly don't include `noDocumentsContent` in the deps array, as it's
-    // causing a visual bug where the "No documents" message is shown for a split second
-    // when clearing a search query with no results
```

Comment:
I can't reproduce this issue mentioned here, so I think we're good.
In any case, it's not ideal that what initially was just `noDocumentsContent` being exempt has eventually led to other deps, `t` and `paneTitle`, being accidentally omitted.
It demonstrates why it's risky to suppress the linter to begin with 😅